### PR TITLE
[internal] Add CI Fix and README to Slack port

### PR DESCRIPTION
**Pull request by iscaltritti**
The functional test we ported is being detected as a regular test in Python's CI, this PR fixes that.
Also adds a README with a pipeline step by step setup guide.

### CODE REVIEW

File: libraries/functional-tests/slacktestbot/README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# Slack functional test pipeline setup
+
+This is a step by step guide to setup the Slack functional test pipeline.
+
+## Slack Application setup
+
+We'll need to create a Slack application to connect with the bot.
+
+1. Create App
```
Comment (sw-joelmut) [inline libraries/functional-tests/slacktestbot/README.md:9]: Not indented properly for each item.

**Review by sw-joelmut — CHANGES_REQUESTED** [review]

File: libraries/functional-tests/slacktestbot/README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+## Azure setup
+
+We will need to create an Azure App Registration and setup a pipeline.
+
+### App Registration
+
+1. Create an App Registration
+
+Go here and click on New Registration.
```
Comment (ceciliaavila) [inline libraries/functional-tests/slacktestbot/README.md]: The link is missing here.

**Review by ceciliaavila — CHANGES_REQUESTED** [review]

**Comment by iscaltritti** [comment]
Thanks @sw-joelmut and @ceciliaavila, fixes applied.

**Review by sw-joelmut — APPROVED** [review]

**Review by ceciliaavila — APPROVED** [review]

File: libraries/functional-tests/slacktestbot/README.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+6. Get the Channel ID
+
+    Go to the Slack workspace you associated the app to. The new App should have appeared; if not, add it using the plus sign that shows up while hovering the mouse over the Apps tab.
+
+    Right click on it and then on Copy link.
+
+    ![ChannelID](./media/SlackChannelID.png)
+
+    The link will look something like *https://workspace.slack.com/archives/**N074R34L1D***.
```
Comment (santgr11) [inline libraries/functional-tests/slacktestbot/README.md]: The intent with this was to get the code in bold? it's looking like this
image

Comment (iscaltritti) [reply libraries/functional-tests/slacktestbot/README.md]: It was, I corrected it just now.

**Pull request merged**